pkg/k8shandler/visualization.go

```diff
@@ -40,46 +44,31 @@ var (
 // CreateOrUpdateVisualization reconciles visualization component for cluster logging
 func (clusterRequest *ClusterLoggingRequest) CreateOrUpdateVisualization(proxyConfig *configv1.Proxy) (err error) {
 	if clusterRequest.cluster.Spec.Visualization == nil || clusterRequest.cluster.Spec.Visualization.Type == "" {
-		if err = clusterRequest.removeKibana(); err != nil {
-			return
+		if err := clusterRequest.removeKibana(); err != nil {
+			logrus.Error(err.Error())
+			return nil
```

Comment:
why are we returning `nil` if we were returning an error before? we should revert this... 
since we have a return of `err error` if we set `err` and then return we are actually doing `return err`